fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -269,6 +280,8 @@ module.exports = function(config, DB) {
               })
               .then(function(result) {
                 t.deepEqual(result, {}, 'Returned an empty object on session token update')
+
+                // Fetch all of the sessions tokens for the account
```

Comment:
:heart: breaking these up with high-level comments, really helps with reading through the flow